users/views.py

```diff
@@ -0,0 +1,77 @@
+import json, bcrypt, jwt, re
+
+from django.views           import View
+from django.http            import JsonResponse
+from django.db              import IntegrityError
+from django.core.exceptions import ObjectDoesNotExist
+
+from gonggol.settings import SECRET_KEY, ALGORITHM
+from .models          import User
+
+class JoinIn(View):
+    def post(self, request):
+        try:
+            data            = json.loads(request.body)
+            account         = data['account']
+            password        = data['password']
+            name            = data['name']
+            postal          = data['postal']
+            address         = data['address']
+            address_detail  = data['address_detail']
+            phone_number    = data['phone_number']
+            sms_reception   = data['sms_reception']
+            email           = data['email']
+            email_reception = data['email_reception']
+
+            email_regex   = '^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$'
+            pw_regex      = '^[A-Za-z0-9@#$!]{8,12}$'
+            account_regex = '^[A-Za-z0-9]{8,12}$'
+
+            if not re.match(account_regex, account): # 이하 형식오류 처리
+                return JsonResponse({"message":"id 형식을 확인하세요."}, status=400)
+            elif not re.match(pw_regex, password):
```

Comment:
elif 가 아닌 if 로 해도 괜찮습니다!
```suggestion
            if not re.match(pw_regex, password):
```

Reply:
수정했습니다!